vie.ini

```
@@ -3,10 +3,442 @@
 ; A message doesn't need to be there to be shown correctly on the client.
 ; Only messages shown in PocketMine itself need to be here
 
+language.name=Tiếng Việt
+language.selected=Đã chọn {%0} ({%1}) là ngôn ngữ cơ bản
 
+ability.flight=Bay
+ability.noclip=No-clip
+chat.type.achievement={%0} has just earned the achievement {%1}
+chat.type.admin=[{%0}: {%1}]
+chat.type.announcement=[{%0}] {%1}
+chat.type.emote=* {%0} {%1}
+chat.type.text=<{%0}> {%1}
+commands.ban.success=Đã cấm người chơi {%0}
+commands.ban.usage=/ban <tên người chơi> [lý do ...]
+commands.banip.invalid=Bạn đã nhập một địa chỉ IP không hợp lệ hoặc người chơi đó không trực tuyến
+commands.banip.success.players=Banned IP address {%0} belonging to {%1}
+commands.banip.success=Đã cấm địa chỉ IP {%0}
+commands.banip.usage=/ban-ip <địa chỉ IP|tên người chơi> [lý do ...]
+commands.banlist.ips=Có tất cả {%0} địa chỉ IP đã bị cấm:
+commands.banlist.players=There are {%0} total banned players:
+commands.banlist.usage=/banlist [ips|players]
+commands.clear.failure.no.items=Could not clear inventory of {%0}, no items to remove
+commands.clear.success=Cleared the inventory of {%0}, removing {%1} items
+commands.clear.testing={%0} has {%1} items that match the criteria
+commands.defaultgamemode.success=The world's default game mode is now {%0}
+commands.defaultgamemode.usage=/defaultgamemode <mode>
+commands.deop.message=You have been de-opped
+commands.deop.success=De-opped {%0}
+commands.deop.usage=/deop <player>
+commands.difficulty.success=Set game difficulty to {%0}
+commands.difficulty.usage=/difficulty <new difficulty>
+commands.effect.failure.notActive.all=Couldn't take any effects from {%0} as they do not have any
+commands.effect.failure.notActive=Couldn't take {%0} from {%1} as they do not have the effect
+commands.effect.notFound=There is no such mob effect with ID {%0}
+commands.effect.success.removed.all=Took all effects from {%0}
+commands.effect.success.removed=Took {%0} from {%1}
+commands.effect.success=Given {%0} * {%1} to {%2} for {%3} seconds
+commands.effect.usage=/effect <player> <effect> [seconds] [amplifier] [hideParticles] OR /effect <player> clear
+commands.enchant.noItem=The target does not hold an item
+commands.enchant.notFound=There is no such enchantment with ID {%0}
```

```suggestion
commands.enchant.notFound=Không có phù phép nào với ID {%0}
```
`enchant` : `phù phép`, `cường hóa`, or `mê hoặc`